components/app/Calendar.vue

```diff
@@ -233,12 +249,27 @@ const groupedEvents = computed(() => {
           class="content-full"
           v-html="selectedEvent.description"
         />
-        <div>
-          <strong>Event details:</strong>
-          <ul>
-            <li>Event starts at: {{ selectedEvent.start.formatTime(TIME_FORMAT) }}</li>
-            <li>Event ends at: {{ selectedEvent.end.formatTime(TIME_FORMAT) }}</li>
-          </ul>
+        <div class="flex  justify-between">
```

Comment:
nit: remove extra space
```suggestion
        <div class="flex justify-between">
```